andela/ah-titans-frontend
# #160168203 Restructure application

#### What does this PR do?
Restructure application structure

#### Description of tasks to be completed
Ensure components are reusable.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout ch-restructure-app-160168203
```
```sh
$ cd ah-titans-frontend
```

>Install the dependencies using the command below.

```sh
$ npm install
```
>Run the application as below.
```sh
$ npm start
```
>Now you can navigate to the link 'https://http://localhost:8080/signup' on your browser and you should be able to see a page as the one shown on the screenshot below.

<img width="1280" alt="screen shot 2018-08-31 at 12 44 34" src="https://user-images.githubusercontent.com/12511966/44905743-af34de80-ad1b-11e8-8013-388a019b0536.png">



#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160168203](https://www.pivotaltracker.com/story/show/160168203)
